diff --git a/node_modules/bingo/lib/cli/loggers/logHelpText.js b/node_modules/bingo/lib/cli/loggers/logHelpText.js
index 5ddc239..b8021f3 100644
--- a/node_modules/bingo/lib/cli/loggers/logHelpText.js
+++ b/node_modules/bingo/lib/cli/loggers/logHelpText.js
@@ -38,6 +38,24 @@ export function logHelpText(mode, from, template) {
             text: 'Whether to run in an "offline" mode that skips network requests.',
             type: "boolean",
         },
+        {
+            examples: ["--remote"],
+            flag: "--remote",
+            text: "Whether to create a remote repository on GitHub if one does not already exist.",
+            type: "boolean",
+        },
+        {
+            examples: ["--skip-files"],
+            flag: "--skip-files",
+            text: "Whether to skip creating files on disk.",
+            type: "boolean",
+        },
+        {
+            examples: ["--skip-requests"],
+            flag: "--skip-requests",
+            text: "Whether to skip sending network requests as specified by templates.",
+            type: "boolean",
+        },
         {
             examples: ["--version"],
             flag: "--version",
